lib/rmt/cli/base.rb

```diff
@@ -90,8 +90,11 @@ def subcommand_prefix
 
   private
 
-  def needs_path(path)
+  def needs_path(path, writable: false)
```

Comment:
What's `writable` argument for? The method is always called with `writable: true`.

Reply:
Ah, I see.